### PR TITLE
Grid, Navigation and Toast Components

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Made grid, navigation and toast components
Look into 
https://vibe-ui-9c30yk5yq-bhargavirdy.vercel.app/docs/toast.html
https://vibe-ui-9c30yk5yq-bhargavirdy.vercel.app/docs/navigation.html
https://vibe-ui-9c30yk5yq-bhargavirdy.vercel.app/docs/grid.html
 for changes

### CODE REVIEW

File: docs/toast.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                        class="fab fa-twitter"></i></a></li>
+            <li class="list-inline-item m-4"><a class="link"
+                    href="https://www.linkedin.com/in/bhargavi-sunkireddy-a60b041a1/"><i
+                        class="fab fa-linkedin-in"></i></a></li>
+        </ul>
+        <div class="text-primary pt-3">© 2022 Bhargavi Sunkireddy</div>
+    </footer>
+    <script src="sidebar.js"></script>
+    <script>
+        const toastSuccessBtn = document.querySelector(".toast-success-btn");
```
Comment (ajit-sharmaGH) [inline docs/toast.html:94]: You can use js in external file

**Review by ajit-sharmaGH — APPROVED** [review]
Looks good, commented something

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.nav-bar{
+    box-sizing: border-box;
+    padding: 0.3rem 1rem;
+    width: 100%;
+    /* height: auto; */
+    /* min-height: 10vh; */
```
Comment (harshitadaswani) [inline components/navigation.css:6]: remove comments

File: components/toast.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    z-index: 5;
+    text-align: right;
+    display: flex;
+    flex-direction: column;
+}
+.toast{
+    justify-content: flex-end;
+    padding: 0.5rem 1rem;
+    width: fit-content;
+    /* position: fixed; */
```
Comment (harshitadaswani) [inline components/toast.css:14]: remove comment

**Review by harshitadaswani — APPROVED** [review]
Looks good!

**Pull request merged**